src/pages/[id]/index.js

```diff
@@ -50,11 +43,21 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
     fillActiveTasksArray();
   }, [fillActiveTasksArray]);
 
+  if (errorMessage) {
+    return <NotFound errorMsg={errorMessage} />;
+  }
+
+  if (!isSuperUser && !userApiCalled) {
+    setUserPrivileges();
+  }
```

Comment:
This is a potential threat that can cause the page to infinitely re-render. I have created a ticket for this https://github.com/Real-Dev-Squad/website-members/issues/419. Will be refactoring it there.